### PR TITLE
chore: my collection design tweaks

### PR DESCRIPTION
**Pull request by brainbicycle**
The type of this PR is: **chore**

This PR resolves [CX-1942]

### Description

- Updates input titles to be capitalized and Text size XS on main form and additional details form
- Updates medium picker to be a radio button

### Screenshots

<details><summary>Before</summary>

MainFormBeforeTweaks
AdditionalDetailsBeforeTweaks

</details>

<details><summary>After</summary>

MainFormAfterTweaks
AdditionalDetailsAfterTweaks

</details>

### Note
- Noticed while I was working on this that the radio button text is not centered is this a known bug?

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

- Update titles + medium select in my collection forms - Brian

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-1942]: https://artsyproduct.atlassian.net/browse/CX-1942

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- iOS user-facing changes (Update titles + medium select in my collection forms - Brian)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 8e4b1f5e1bd92e3302298ff2bb38eb5ee054b7ad
</p>

File: src/lib/Components/TextArea.tsx [hunk 1]
```diff
@@ -17,18 +18,25 @@ export const TextArea: React.FC<TextAreaProps> = ({ title, ...props }) => {
...
+
   return (
     <>
-      {!!title && (
-        <Text mb={1} variant="mediumText">
-          {title}
-        </Text>
-      )}
+      {!!title &&
+        (allowV3 ? (
```
Comment (brainbicycle) [inline src/lib/Components/TextArea.tsx:26]: similar to above comment not sure if I should branch or just replace

File: src/lib/Scenes/MyCollection/Screens/ArtworkFormModal/Screens/MyCollectionArtworkFormMain.tsx [hunk 2]
```diff
@@ -48,12 +48,11 @@ export const MyCollectionArtworkFormMain: React.FC<StackScreenProps<ArtworkFormM
...
       <ScrollView keyboardDismissMode={"on-drag"} keyboardShouldPersistTaps={"handled"}>
         <Spacer my={1} />
-
-        <Sans size="4" textAlign="center">
+        <Text textAlign="center">
           {addOrEditLabel} details about your artwork to access {"\n"}
           price and market insights.
-        </Sans>
-
+        </Text>
```
Comment (pvinis) [inline src/lib/Scenes/MyCollection/Screens/ArtworkFormModal/Screens/MyCollectionArtworkFormMain.tsx:54]: love it!

File: src/lib/Scenes/MyCollection/Screens/ArtworkFormModal/Components/ArtistAutosuggest.tsx [hunk 3]
```diff
@@ -17,17 +17,15 @@ export const ArtistAutosuggest: React.FC = () => {
...
+          <Text size="xs">ARTIST</Text>
           <Spacer mt={0.5} />
           <ArtistSearchResult result={artistSearchResult} />
         </>
       ) : (
         <SearchContext.Provider value={searchProviderValues}>
           <Box>
             <Input
-              title="Artist"
+              title="ARTIST"
```
Comment (pvinis) [inline src/lib/Scenes/MyCollection/Screens/ArtworkFormModal/Components/ArtistAutosuggest.tsx:28]: I guess we could make this work in the Input component, but cool for now.

File: src/lib/Components/Input/InputTitle.tsx [hunk 4]
```diff
@@ -7,16 +8,33 @@ export const InputTitle: React.FC<{ required?: boolean }> = ({ children: title,
...
-          <Text variant="mediumText" color={color("purple100" /* TODO-PALETTE-V3 "blue100" */)}>
-            *
-          </Text>
-        )}
-      </Text>
-    </ThemeV2>
-  )
+  const allowV3 = usePaletteFlagStore((state) => state.allowV3)
+
+  if (allowV3) {
```
[the remark this answers is not in the document]
Comment (pvinis) [reply src/lib/Components/Input/InputTitle.tsx:13]: That's good. You could totally remove the v2, but I can do it in the cleanup PR. 👍

File: src/lib/Components/Input/InputTitle.tsx [hunk 5]
```diff
@@ -7,16 +8,33 @@ export const InputTitle: React.FC<{ required?: boolean }> = ({ children: title,
...
+    return (
+      <ThemeV3>
+        <Text size="xs" mb={0.5}>
+          {title}
```
Comment (pvinis) [inline src/lib/Components/Input/InputTitle.tsx:17]: Here is the place, we could do like `title.toUpperCase()`. Since the app is English only, we won't have any problems, but we can do it in another PR.

**Review by pvinis — APPROVED** [review]
